packages/multichain-transactions-controller/jest.config.js

```diff
@@ -17,8 +17,8 @@ module.exports = merge(baseConfig, {
   // An object that configures minimum threshold enforcement for coverage results
   coverageThreshold: {
     global: {
-      branches: 95,
-      functions: 97,
+      branches: 90,
+      functions: 95,
       lines: 97,
       statements: 97,
```

Comment:
We can also revert this and increase the threshold now that everything is fully tested 💪 

```suggestion
      branches: 100,
      functions: 100,
      lines: 100,
      statements: 100,
```